src/home-assistant.html

```diff
@@ -261,5 +265,11 @@
       this.handleConnectionPromise(this.connectionPromise);
     }
   },
+
+  setTheme: function (event) {
+    this.hass.selectedTheme = event.detail;
```

Comment:
Please don't assign to the hass object directly but instead use Object.assign to make a copy. This should be done for the other theme related functions too.

Example code from this file:

```js
this.hass = Object.assign({}, this.hass, { config: config });
```